opennms-model/src/main/java/org/opennms/netmgt/model/topology/BridgeForwardingTable.java

```diff
@@ -0,0 +1,253 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2014 The OpenNMS Group, Inc.
```

Comment:
New source files should use the current year in the copyright header.

Reply:
ok